src/app/redux/FetchDataSaga.js

```diff
@@ -39,7 +39,6 @@ export function* fetchState(location_change_action) {
         const username = m[1];
         yield fork(fetchFollowCount, username);
         yield fork(loadFollows, 'getFollowersAsync', username, 'blog');
-        yield fork(loadFollows, 'getFollowingAsync', username, 'blog');
```

Comment:
We need to keep this line

Reply:
Why do we need to do this twice?

Reply:
at a glance I thought it was the same thing also, but it is getFollowers and getFollowing

Reply:
OH MAN. Thank you. Rolling back the change